### ISSUE TITLE
MAX32655FTHR image indetifes as MAX78000

### ISSUE THREAD

**Issue by misaz** [MaximIntegrated/max32625pico-firmware-images#5]
I bought MAX32655FTHR in Mouser ELectronics yesterday. Stock DAPLINK did not work with OpenOCD well and it always failed with error

```
target max32xxx.cpu examination failed
```

I decided to upgrade firmware of DAPLINK. Downloaded `max32625_max32655fthr_if_crc_swd_v1.0.3.bin` (MD5: 7686086b3f2fdce852c0ecf060ce9f75), booted MAX32655FTHR to MAINTANCE mode and flashed the new firmware to the device. New DAPLINK firmware flashed successfully but next time it appeared as a daplink of MAX78000 instead of MAX32655. Can you check that image uploaded as `max32625_max32655fthr_if_crc_swd_v1.0.3.bin` is correct image for MAX32655FTHR?

My issue with examination resolved after flashing this new firmware and now I can use MAX32655FTHR with OpenOCD but I am facing other issue. Every 2 seconds when CPU is not halted it reports polling error:

![Polling target failed](https://user-images.githubusercontent.com/2937706/219216467-e26363d8-c815-43d9-9a33-09141659aee2.png)

To summarize I have two questions:

1) Is the image `max32625_max32655fthr_if_crc_swd_v1.0.3.bin` is correct image for MAX32655FTHR?
2) What shoudl I do to get rid of polling errors?
